src/SeleniumLibrary/keywords/browsermanagement.py

```diff
@@ -239,21 +239,41 @@ def get_location(self):
         return self.driver.current_url
 
     @keyword
-    def location_should_be(self, url):
-        """Verifies that current URL is exactly ``url``."""
+    def location_should_be(self, url, message=None):
+        """Verifies that current URL is exactly ``url`.
+
+        The ``url`` argument contains the exact url that should exist in browser.
+
+        The ``message`` argument can be used to override the default error
+        message.
+
+        ``message`` argument new in SeleniumLibrary 3.2.0.
+        """
         actual = self.get_location()
         if actual != url:
-            raise AssertionError("Location should have been '%s' but was "
-                                 "'%s'." % (url, actual))
+            if is_noney(message):
+                message = "Location should have been '%s' but "\
```

Comment:
Splitting strings with `\` creates code which is easy to break. It would be better to do it like this: 
```
message = ("Location should have been '%s' but "
           "was '%s'." % (url, actual))
```

Please also fix line 274 and 275.